quadra/tasks/classification.py

```diff
@@ -227,7 +230,21 @@ def model(self, model_config: DictConfig) -> None:
             model_config.model, classifier=self.classifier, pre_classifier=self.pre_classifier, _convert_="partial"
         )
         if getattr(self.config.backbone, "freeze_parameters_name", None) is not None:
-            self.freeze_layers(self.config.backbone.freeze_parameters_name)
+            self.freeze_layers_by_name(self.config.backbone.freeze_parameters_name)
+
+        if getattr(self.config.backbone, "freeze_parameters_index", None) is not None:
```

Comment:
Should we also check if both `freeze_parameters_name` and `freeze_parameters_index` are not None?

Reply:
What would you do if they are both None? Right now it just doesn't do anything which I would expect is the desired behaviour

Reply:
I also think it's ok to let it freeze both by name and index. Maybe we could add a warning to inform that you are using both. Because maybe you left one type of freeze in the basic experiment config

Reply:
Ooops I mistook reading, mmm ok, so right it will first freeze by name and then freeze by index, I agree on adding a warning

Reply:
Yes, warning would be just fine also for me.